client/src/components/settings/__snapshots__/honesty.test.tsx.snap

```diff
@@ -8,7 +8,7 @@ exports[`<Honesty /> <Honesty /> snapshot when isHonest is false: Honesty 1`] =
     settings.headings.honesty
   </SectionHeader>
   <FullWidthRow>
-    <Uncontrolled(Panel)
+    <d
```

Comment:
This is a bit weird, but I think it's a problem for another day.

Reply:
Yeah, I think it's mimicing what visual dom look like, not the actual dom, this is how the component look in react dev tools